frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx

```diff
@@ -2,17 +2,21 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
+import { compose } from "redux";
 
 import { t } from "ttag";
-import _ from "underscore";
+import { isObject, reject } from "lodash";
```

Comment:
It was cancelled since the easiest path is just upgrading Underscore (PR #22705).
Not sure if the current Underscore already has `isObject` and `reject`.

Reply:
- https://underscorejs.org/#isObject
- https://underscorejs.org/#reject

Reply:
Just to be clear, are we not going to use `lodash` anymore? If so, I guess I should revert the change in this PR back to using `underscore`. And eventually `lodash` will also be removed from the dependency.

Right?

Reply:
It's not supercritical (both libs are small) but whenever possible, just stay with Underscore.